services/quiz.js

```diff
@@ -109,14 +117,26 @@ function generateConjugation(verb, pronoun, moodTense) {
 	return null;
 }
 
-function generateQuestion() {
+function generateQuestion(verbSet) {
 	const moodTense = chooseMoodTense();
 	const pronoun = choosePronoun();
-	const verb = chooseVerb();
+	const data = verbSets[verbSet];
+	let verbs=[];
+	let hasDefinition = false;
+	if (data.constructor === Array){
```

Comment:
@sdee I don't like this approach (hacky). Very non-intuitive way of organizing the verb sets. I recommend on deciding on a format in the verb files and sticking with it.